dmapiclient/data.py

```diff
@@ -140,6 +140,12 @@ def register_framework_interest(self, supplier_id, framework_slug, user):
             user=user,
         )
 
+    def find_supplier_declarations(self, supplier_id):
+        response = self._get(
+            "/suppliers/{}/frameworks".format(supplier_id)
+        )
+        return {'declarations': response['frameworkInterest']}
```

Comment:
We don't generally have the ApiClient muddy the API response at all - we just pass the response as-is. Looks like the only time we do wrap things I a new key is in `get_supplier_declaration`, which I suspect is where you got the idea to do the same thing here. I don't think that's a practice we want to spread though, and this method would be better just passing the `frameworkInterest` response through unmodified.